### PR TITLE
Confirm dataset deletion, email finetune success, UI fixes.

### PR DESCRIPTION
**Pull request by Kovbo**
Task: Require users to confirm by typing the dataset name before deleting a dataset.

UI:
datasetDelete

Task: Send success emails for fine tune creation.

UI:
email

Task: Avoid models table overflow in usage view.
UI + responsive improvements:
usage-mobile

Task: Usage numbers should always include two decimals
Done.

### CODE REVIEW

File: app/src/server/tasks/fineTuning/checkFineTuneStatus.task.ts [hunk 1]
```diff
@@ -83,6 +85,20 @@ export const checkFineTuneStatus = defineTask({
               },
             });
 
+            // Notify the owner that the model has been trained
+            const project = await prisma.project.findUniqueOrThrow({
+              where: { id: typedFT.projectId },
+            });
+            await sendToOwner(typedFT.projectId, (email: string) =>
```
Comment (Kovbo) [inline app/src/server/tasks/fineTuning/checkFineTuneStatus.task.ts:92]: Do we notify a project owner or all admins?

Comment (corbt) [reply app/src/server/tasks/fineTuning/checkFineTuneStatus.task.ts:92]: Actually let's track the user who created the fine-tune and alert just that user in this case. Shouldn't be that hard to add and will be a better ux.

Comment (Kovbo) [reply app/src/server/tasks/fineTuning/checkFineTuneStatus.task.ts:92]: I can do it by adding a column to the fineTunes table in order to track a user.
Do we need to backfill the existing models with users?

Comment (corbt) [reply app/src/server/tasks/fineTuning/checkFineTuneStatus.task.ts:92]: Nah let's just add the column going forward, no need to backfill.

File: app/src/server/emails/templates/layout/Highlight.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Section } from "@react-email/components";
+import React, { CSSProperties, ReactNode } from "react";
+
+interface Props {
+  style?: CSSProperties;
+  children: ReactNode;
+}
+export default function Headlight({ style, children }: Props) {
```
Comment (arcticfly) [inline app/src/server/emails/templates/layout/Highlight.tsx]: Nit: Headlight -> Highlight

File: app/src/server/emails/templates/layout/InvoiceDescription.tsx [hunk 3]
```diff
@@ -1,6 +1,7 @@
-import { Column, Row, Section } from "@react-email/components";
+import { Column, Row } from "@react-email/components";
 import React, { CSSProperties } from "react";
 import { Text } from ".";
+import Headlight from "./Highlight";
```
Comment (arcticfly) [inline app/src/server/emails/templates/layout/InvoiceDescription.tsx]: Headlight -> Highlight

**Review by arcticfly — APPROVED** [review]

**Pull request merged**